apps/doji-frontend/src/pages/schedule-session/index.tsx

```diff
@@ -0,0 +1,268 @@
+})
+
+type scheduleModel = yup.InferType<typeof createScheduleValidation>
+
+export function Index() {
+  const [openDialog, setOpenDialog] = React.useState(false)
+  const [confirm, setConfirm] = React.useState(false)
+  const [serviceData, setServiceData] = React.useState<IServiceInformationDTO>(null)
+  const [scheduleSessionData, setscheduleSessionData] = React.useState<IScheduleSessionDTO>({
+    fee: 0,
+    expertUsername: '',
+    serviceName: '',
+    duration: 0,
+    startTime: new Date(),
+    participantsUsername: [],
+  })
+  const {
+    register,
+    watch,
+    handleSubmit,
+    formState: { errors },
+    control,
+  } = useForm<scheduleModel>({
+    resolver: yupResolver(createScheduleValidation),
+    defaultValues: {
+      date: new Date(),
+      startTime: new Date(),
+      endTime: new Date(),
+      participants: [],
+    },
+  })
+  const watchAll = watch()
+  useEffect(() => {
+    const param = new URLSearchParams(window.location.search)
+    scheduleSessionData.expertUsername = param.get('expert_username')
+    scheduleSessionData.serviceName = param.get('service_name')
+    const url =
+      'http://localhost:3333/api/session/service/' +
+      param.get('expert_username') +
+      '/' +
+      param.get('service_name')
+    axios.get(url).then((res) => {
+      setServiceData(res.data)
+    })
+  }, [])
+  function handleOpenDialog() {
+    setOpenDialog(true)
+  }
+
+  function handleCloseDialog(value) {
+    setOpenDialog(false)
+    setConfirm(value)
+    if (value) {
+      console.log(scheduleSessionData)
+    }
+  }
+  function selectedTagsHandler(items) {
+    console.log(items)
+  }
+  function calculateTotal() {
+    const timeDiff = watchAll.endTime.getTime() - watchAll.startTime.getTime()
+    const duration = Math.round((timeDiff * 10) / 36e5) / 10
+    return duration * serviceData.fee * (watchAll.participants.length + 1)
+  }
+  const onSubmit: SubmitHandler<scheduleModel> = async (data) => {
+    handleOpenDialog()
+    const timeDiff = data.endTime.getTime() - data.startTime.getTime()
+    scheduleSessionData.duration = Math.round((timeDiff * 10) / 36e5) / 10
+    const startDate = new Date(data.date)
+    startDate.setHours(
+      data.startTime.getHours(),
+      data.startTime.getMinutes(),
+      data.startTime.getSeconds(),
+    )
+    scheduleSessionData.startTime = startDate
+    scheduleSessionData.fee =
+      scheduleSessionData.duration * serviceData.fee * (data.participants.length + 1)
+    scheduleSessionData.participantsUsername = data.participants.map((element) => {
+      return element.value
+    })
+  }
+  if (serviceData === null) {
+    return null
+  }
+  return (
+    <Box display="flex" flexDirection="column" position="relative" minHeight="sm">
+      <TopBar title="New session" action={TopBarActionType.Back}></TopBar>
+      <Box>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <Grid container alignItems="center">
+            <Grid item xs={12}>
```

Comment:
Actually, I dont recommend you to use `Grid` since it will mess up the code. 

For this page, I think only `Stack` (e.g. flex) is enough